### DISCUSSION TITLE
EZVIZ C2C remove TIM Firmware

### DISCUSSION THREAD

**Discussion by mschiavo** [indomus/forum#973] [Hardware & Firmware]
Hi all, 
I've two exwiz c2c mini cam with the Italian TIM firmware and I can't use it with the ezviz app (so I can't integrate with IFTTT ore Google Home). I only can use with a TIM app.
Is there any way to flash a new ezviz firmware? 

Thanks a lot.

Marco

**Comment by indomus** [comment]
In italiano Marco, per favore.

**Reply by mschiavo** [reply]
Perfetto! Meglio ancora! 

Vi spiego il mio problema.
Con il nuovo abbonamento alla fibra di Tim ho incluso nella promozione due ipcam. 
Le cam sono le Ezviz C2C. Per il mio uso vanno benissimo solo che sono state marchiate TIM e possono essere usate solo con l'app IoTim che non è che permetta grandi cose.
Purtroppo non possono essere usate con l'app ezviz e non riesco quindi a connetterle con Google Home (le vede ma non le può usare) o con IFTTT. 
Esiste un qualche modo per flashare un firmware stock o qualcosa di simile? 

Grazie mille a tutti.

**Comment by Cusu95** [comment]
Io ho integrato le Tim cam con flusso rstp seguendo questa guida:
 **LINK RIMOSSO**

**Reply by snakuzzo** [reply]
Non sono consentiti link a siti terzi. Rimuovo

**Comment by Cusu95** [comment]
Mi scuso per il link.
Per integrarle, prima le imposti con l’app delle timcam (ioTim mi pare si chiami).
Dopodiché, aggiungi in configurazione.yaml le seguenti impostazioni:
camera:
  platform: ffmpeg
  name: NomeTelecamera
  input: rtsp://admin:SECRET@IP:554/Streaming/Channels/101

NomeTelecamera è il nome che apparirà quando si visiona il video da Home Assistant
SECRET corrisponde al codice di sei lettere maiuscole presente sull'etichetta sotto alla videocamera
IP corrisponde all'indirizzo IP assegnato alla telecamera

**Reply by snakuzzo** [reply]
Sono stufo di ripeterlo...dovete rispondere sotto l'ultimo messaggio, non creare nuovi thread nella stessa discussione.... non ci vuole molto.
Grazie

**Reply by mschiavo** [reply]
> Mi scuso per il link. Per integrarle, prima le imposti con l’app delle timcam (ioTim mi pare si chiami). Dopodiché, aggiungi in configurazione.yaml le seguenti impostazioni: camera: platform: ffmpeg name: NomeTelecamera input: rtsp://admin:SECRET@IP:554/Streaming/Channels/101
> 
> NomeTelecamera è il nome che apparirà quando si visiona il video da Home Assistant SECRET corrisponde al codice di sei lettere maiuscole presente sull'etichetta sotto alla videocamera IP corrisponde all'indirizzo IP assegnato alla telecamera

Buongiorno! 
Mi dici che l’unico modo alternativo è quello di mettere in piedi tutto il sisitema di controllo domotico di Home Assistant; configurare al suo interno tutta la domotica di casa, in modo da poter avere un uso integrato delle due cam? 
Non esiste altri modo per farle interagire con una casa già domotica? 
Grazie mille!

**Reply by Cusu95** [reply]
Il flusso rtsp può essere integrato in quasi ogni sistema domotico. Può essere visualizzato banalmente anche con VLC.
L’importante è impostare le telecamere con l’app in modo che siano funzionanti. Poi bisogna assegnare un indirizzo ip statico, e a quel punto con i dati sotto la telecamera completi le informazioni per il flusso rtsp. Quel flusso puoi inserirlo nel tuo sistema preferito di domotica

**Reply by mschiavo** [reply]
Perfetto! 
Sono riuscito ad integrarle!
Su Home Assistant posso gestire anche gli eventi di riconoscimento movimento che la cam ha a disposizione?
